docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -745,10 +745,9 @@ Possible values are:
 | Network   | Chain   | Type        | Description                                                         |
 |-----------|---------|-------------|---------------------------------------------------------------------|
 | `mainnet` | Eth 2.0 | Production  | Main network.                                                       |
-| `altona`  | Eth 2.0 | Test        | Multi-client testnet maintained by the Goerli testnet initiative.   |
-| `medalla` | Eth 2.0 | Test        | Multi-client testnet maintained by the Goerli testnet initiative.   |
+| `medalla` | Eth 2.0 | Test        | Multi-client testnet.                                               |
```

Comment:
Should we add something like "retired" in front of this testnet?

Reply:
I think we could just remove medalla at this point.  The network is rapidly being abandoned and we'll likely shutdown our nodes in the next week or two anyway.